docs/guides/solr/README.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Solr
+menu:
+  docs_{{ .version }}:
+    identifier: sl-readme-solr
+    name: Solr
+    parent: sl-solr-guides
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+url: /docs/{{ .version }}/guides/solr/
+aliases:
+  - /docs/{{ .version }}/guides/solr/README/
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
```

Comment:
Add a overview block like this -

# Overview 

SingleStore, a distributed SQL database for real-time analytics, transactional workloads, and operational applications. With its in-memory processing and scalable architecture, SingleStore enables organizations to achieve high-performance and low-latency data processing across diverse data sets, making it ideal for modern data-intensive applications and analytical workflows.